src/StarcounterClientFiles/wwwroot/sys/app-shell/app-shell-without-service-worker.html

```diff
@@ -0,0 +1,39 @@
+﻿<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8">
+    <title>{0}</title>
+    <script>  
+      (async function() {{
+        const logPrefix = '[Starcounter App Shell]';
+        if ('serviceWorker' in navigator) {{
+          const registrations = await navigator.serviceWorker.getRegistrations();
+          for(const registration of registrations) {{
+              registration.unregister()
+          }}
+          if(registrations.length > 0) {{
+            console.info(`${{logPrefix}} Removed ${{registrations.length}} service worker registration${{registrations.length > 1 ? 's' : ''}}`);
+            // this `debugger` is left here on purpose.
+            // To allow developers (who have DevTools open) pause here and see if any registrations removals occurred before auto-reload is triggered
```

Comment:
Do you think there is such a big benefit to know that? I'd rather have `Incomment the below debugger statement, to see if any registrations removals occurred before auto-reload is triggered`. The code in this file should be production ready for releasing to end-users.